test_random.js

```diff
@@ -0,0 +1,13 @@
+//const random = require("../node_modules/random-js").Random;
```

Comment:
Please use this file for new class Utils

Reply:
use `static` methods

Reply:
fixed